app/create_app.py

```diff
@@ -110,13 +113,31 @@ def ensure_minimum_required_roles():
                 unprotected_routes=["/", "/healthcheck"],
             )
 
+        # Get static filenames list
+        static_files_list = []
+        for _, _, files in os.walk(flask_app.static_url_path):
+            for file in files:
+                static_files_list.append(file)
+
         @flask_app.after_request
         def set_response_headers(response):
-            response.headers[
-                "Cache-Control"
-            ] = "no-cache, no-store, must-revalidate"
-            response.headers["Pragma"] = "no-cache"
-            response.headers["Expires"] = "0"
+            filename = (
+                re.findall(
+                    "filename=(.+)", response.headers["Content-Disposition"]
```

Comment:
nice use of re. Should we be checking for "inline" only here?

Reply:
yes. ```
response.headers.get("Content-Disposition") = 
'inline; filename=govuk-frontend-4.0.0.min.css'
```
trying extract filename from here

Reply:
I think it's a good point, but should we cache when it's "attachment" also ?

Reply:
yeah, i was just checking but i think it makes sense to cache both to be fair